class-astra-theme-customizer-reset.php

```diff
@@ -116,6 +116,9 @@ public function ajax_customizer_reset() {
 				// Delete Global Color Palette and Typography Preset options.
 				delete_option( 'astra-typography-presets' );
 				delete_option( 'astra-color-palettes' );
+
+				// Delete Astra's 4.0.0 admin setting options.
+				delete_option( 'astra_admin_settings' );
```

Comment:
Should we also reset mega menu options ? , Just a thought

Reply:
No actually, menu is totally different part. It comes under wp-core, megamenu options store in postmeta DB, so they can handle separately. Here we are only clearing theme's option table keys.